Sources/StreamChatUI/ChatMessageList/ChatMessageListKeyboardObserver.swift

```diff
@@ -7,23 +7,20 @@ import UIKit
 
 open class ChatMessageListKeyboardObserver {
     public weak var containerView: UIView!
```

Comment:
nit: `weak var containerView: UIView!` == `unowned var containerView: UIView` :)

Reply:
the keyboard observer is exposed on `ChatMessageListVC/ChatThreadVC` and has all fields publicly available -> having extra strong ref will make the lifetime longer and cause a crash if the `containerView` is accessed after the deallocation. Made it optional to align with other properties but it'd be nice to get back to the observer one time